cmd/kubeadm/app/cmd/phases/kubelet_test.go

```diff
@@ -20,16 +20,13 @@ import (
 	"testing"
 
 	"github.com/spf13/cobra"
-
```

Comment:
hm, is this something the new gofmt does - i.e. removing these lines?

Reply:
yes...